extensions/tyme-3-time-tracker/src/start-task-tracking.tsx

```diff
@@ -0,0 +1,53 @@
+import { Action, ActionPanel, Icon, List, closeMainWindow } from "@raycast/api";
+import { Task, getTasks, startTrackingTask } from "./tasks";
+import { useCachedPromise, useFrecencySorting } from "@raycast/utils";
+
+function getTaskPath(task: Task): string {
+  return [task.category?.name, task.project.name, task.parentTask?.name]
+    .filter((segment): segment is string => segment !== undefined)
+    .join(" > ");
+}
+
+function StartTrackerForTask() {
+  // Get all tasks
+  const { data: tasks, isLoading: tasksIsLoading } = useCachedPromise(() => getTasks(), [], {
+    keepPreviousData: true,
+  });
+
+  // Sort by frequency
+  const { data: sortedTasks, visitItem: visitTask, resetRanking: resetTaskRanking } = useFrecencySorting(tasks);
+
+  // Handle start tracking
+  const handleStartTracking = (task: Task) => {
+    closeMainWindow({ clearRootSearch: true });
```

Comment:
Maybe add a HUD that it started tracking?

Reply:
Good idea!